imagedephi/web_static/assets/index-1ca2e065.js

```diff
@@ -0,0 +1 @@
s;s=ms(s);for(let r=0;r<fs.length;r++){const o=fs[r]+s;if(o in e)return Xt[t]=o}return t}const us="http://www.w3.org/1999/xlink";function hl(e,t,n,s,r){if(s&&t.startsWith("xlink:"))n==null?e.removeAttributeNS(us,t.slice(6,t.length)):e.setAttributeNS(us,t,n);else{const o=_r(t);n==null||o&&!_s(n)?e.removeAttribute(t):e.setAttribute(t,o?"":n)}}function pl(e,t,n,s,r,o,i){if(t==="innerHTML"||t==="textContent"){s&&i(s,r,o),e[t]=n??"";return}const f=e.tagName;if(t==="value"&&f!=="PROGRESS"&&!f.includes("-")){e._value=n;const d=f==="OPTION"?e.getAttribute("value"):e.value,m=n??"";d!==m&&(e.value=m),n==null&&e.removeAttribute(t);return}let u=!1;if(n===""||n==null){const d=typeof e[t];d==="boolean"?n=_s(n):n==null&&d==="string"?(n="",u=!0):d==="number"&&(n=0,u=!0)}try{e[t]=n}catch{}u&&e.removeAttribute(t)}function gl(e,t,n,s){e.addEventListener(t,n,s)}function ml(e,t,n,s){e.removeEventListener(t,n,s)}const as=Symbol("_vei");function _l(e,t,n,s,r=null){const o=e[as]||(e[as]={}),i=o[t];if(s&&i)i.value=s;else{const[f,u]=bl(t);if(s){const d=o[t]=wl(s,r);gl(e,f,d,u)}else i&&(ml(e,f,i,u),o[t]=void 0)}}const ds=/(?:Once|Passive|Capture)$/;function bl(e){let t;if(ds.test(e)){t={};let s;for(;s=e.match(ds);)e=e.slice(0,e.length-s[0].length),t[s[0].toLowerCase()]=!0}return[e[2]===":"?e.slice(3):Ye(e.slice(2)),t]}let Zt=0;const xl=Promise.resolve(),yl=()=>Zt||(xl.then(()=>Zt=0),Zt=Date.now());function wl(e,t){const n=s=>{if(!s._vts)s._vts=Date.now();else if(s._vts<=n.attached)return;ie(vl(s,n.value),t,5,[s])};return n.value=e,n.attached=yl(),n}function vl(e,t){if(P(t)){const n=e.stopImmediatePropagation;return e.stopImmediatePropagation=()=>{n.call(e),e._stopped=!0},t.map(s=>r=>!r._stopped&&s&&s(r))}else return t}const hs=/^on[a-z]/,El=(e,t,n,s,r=!1,o,i,f,u)=>{t==="class"?fl(e,s,r):t==="style"?al(e,n,s):At(t)?dn(t)||_l(e,t,n,s,i):(t[0]==="."?(t=t.slice(1),!0):t[0]==="^"?(t=t.slice(1),!1):Ol(e,t,s,r))?pl(e,t,s,o,i,f,u):(t==="true-value"?e._trueValue=s:t==="false-value"&&(e._falseValue=s),hl(e,t,s,r))};function Ol(e,t,n,s){return s?!!(t==="innerHTML"||t==="textContent"||t in e&&hs.test(t)&&A(n)):t==="spellcheck"||t==="draggable"||t==="translate"||t==="form"||t==="list"&&e.tagName==="INPUT"||t==="type"&&e.tagName==="TEXTAREA"||hs.test(t)&&z(n)?!1:t in e}const Cl=Z({patchProp:El},il);let ps;function Tl(){return ps||(ps=Lo(Cl))}const Pl=(...e)=>{const t=Tl().createApp(...e),{mount:n}=t;return t.mount=s=>{const r=Il(s);if(!r)return;const o=t._component;!A(o)&&!o.render&&!o.template&&(o.template=r.innerHTML),r.innerHTML="";const i=n(r,!1,r instanceof SVGElement);return r instanceof Element&&(r.removeAttribute("v-cloak"),r.setAttribute("data-v-app","")),i},t};function Il(e){return z(e)?document.querySelector(e):e}const Al=(e,t)=>{const n=e.__vccOpts||e;for(const[s,r]of t)n[s]=r;return n},Ml={};function Fl(e,t){return null}const Rl=Al(Ml,[["render",Fl]]);async function Nl(e){return(await fetch("/directory/"+(e||""),{method:"GET",mode:"cors"})).json()}Pl(Rl).mount("#app");Nl();
```

Comment:
Did you mean to check in these files in the web_static/assets directory?  It looks like the .gitignore meant to ignore them but has an absolute path instead of a relative path.

Reply:
fixed with and amended rebase and forced push